cosigned/base/mutatingwebhook.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
```

Comment:
cluster resource, should go in: 

https://github.com/operate-first/apps/tree/master/cluster-scope/base
(you'll need to create a new folder here, following the same format as the other resources, see readme [here](https://github.com/operate-first/apps/tree/master/cluster-scope) for more info)